packages/ember-htmlbars/lib/utils/extract-positional-params.js

```diff
@@ -25,9 +25,10 @@ function processNamedPositionalParameters(renderNode, positionalParams, params,
 
   for (let i = 0; i < limit; i++) {
     let param = params[i];
+    let isActiveStreamParam = isStream(param) && param.isActive;
 
     assert(`You cannot specify both a positional param (at position ${i}) and the hash argument \`${positionalParams[i]}\`.`,
-           !(positionalParams[i] in attrs));
+           isActiveStreamParam || !(positionalParams[i] in attrs));
```

Comment:
what is this assertion against active doing? it would skip all bound positional params?